src/dev/license_checker/config.ts

```diff
@@ -78,6 +78,18 @@ export const DEV_ONLY_LICENSE_WHITELIST = ['MPL-2.0'];
 
 // Globally overrides a license for a given package@version
 export const LICENSE_OVERRIDES = {
+  // The following packages doesn't have a LICENSE file in their distribution,
+  // but are all forks from another project which at the time of the fork was
+  // licensed with a propriettary Facebook license [1]. Later these projects
+  // have all coverted to MIT. For details see [2].
+  //
+  // 1: https://github.com/graphql/graphql-language-service/blob/a80c0a5035c0bbd57e5c1fd7e2b64d65013a6f66/LICENSE
+  // 2: https://github.com/apollographql/apollo-tooling/issues/1553#issuecomment-571303621
+  '@apollgraphql/graphql-language-service-interface@2.0.2': ['Facebook Propriettary License'],
```

Comment:
I'm not really sure if we should write `MIT` or use the Facebook license as they are kind of in-between. If we use the Facebook license, I'm not sure if what I wrote here is ok, or if it should be written differently.